goblin/driver/cluster.py

```diff
@@ -60,12 +59,13 @@ class Cluster:
         'min_conns': 1,
         'max_times_acquired': 16,
         'max_inflight': 64,
-        'message_serializer': 'goblin.driver.GraphSON2MessageSerializer'
+        'message_serializer': 'goblin.driver.GraphSON2MessageSerializer',
+        'provider': provider.TinkerGraph
     }
```

Comment:
Maybe here they should both be strings instead of a string and an object. Or both objects.